packages/wallets/wallet-wallet-connect/src/strategy/strategy.ts

```diff
@@ -321,7 +317,7 @@ export class WalletConnect
     try {
       this.provider = await EthereumProvider.init({
         projectId: this.metadata.projectId as string,
-        metadata: this.metadata
+        metadata: this
```

Comment:
Are we sure this works? Previously it was `this.metadata.metadata` now it's `this.metadata`